_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -1196,15 +1190,13 @@
 /turf/open/floor/plasteel,
 /area/shuttle/arrival)
 "acu" = (
-/obj/item/weapon/twohanded/required/kirbyplants{
-	icon_state = "plant-21";
-	layer = 4.1;
-	pixel_x = -3;
-	pixel_y = 3
-	},
 /obj/machinery/ai_status_display{
 	pixel_x = 32
 	},
+/obj/item/weapon/twohanded/required/kirbyplants{
+	icon_state = "plant-21";
+	layer = 4.1
```

Comment:
you can safely remove all of the layer definitions for these, since they have a layer already